.github/workflows/build.yml

```diff
@@ -0,0 +1,327 @@
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: metanorma-linux
+        path: build/metanorma
+
+  linux-test:
+    needs: linux-packed-mn
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.ignore-errors }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-16.04, ubuntu-18.04, ubuntu-20.04 ]
+        ignore-errors: [ false ]
+        test-flavor:
+          - iso
+          - cc
+          - un
+          - iec
+          - iho
+          - nist
+          - m3aawg
+          - mpfa
+          - jcgm
+          - csa
+          - ribose
+          - bipm
+          - itu
+          - ietf
+        include:
+          # https://github.com/metanorma/mn-samples-ogc/issues/119
+          - test-flavor: ogc
+            ignore-errors: true
+            os: ubuntu-16.04
+          - test-flavor: ogc
+            ignore-errors: true
+            os: ubuntu-18.04
+          - test-flavor: ogc
+            ignore-errors: true
+            os: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: metanorma/packed-mn
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: metanorma-linux
+          path: build
+
+      - run: chmod +x build/metanorma
+
+      - run: make test-flavor
+        env:
+          GITHUB_CREDENTIALS: "metanorma-ci:${{ secrets.METANORMA_CI_PAT_TOKEN }}"
+          TEST_FLAVOR: ${{ matrix.test-flavor }}
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: site-linux
+          path: site/**
+
+  macos-packed-mn:
+    if: ${{ !startsWith(github.ref, 'refs/tags/') }}
+    needs: macos
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: metanorma/packed-mn
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler: 2.2.3
+          bundler-cache: true
+
+      - run: bundle update --all
+
+      - uses: shogo82148/actions-setup-perl@v1
+        with:
+          perl-version: '5.16.2'
+
+      - run: |
+          curl -Lks 'https://git.io/rg-ssl' | ruby
+          echo "perl -v: $(perl -v)"
+          echo "clang -v: $(clang -v)"
+          echo "clang++ -v: $(clang -v)"
+
+      - run: brew install squashfs
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: rubyc-darwin-x64
+
+      - run: mv rubyc-darwin-x64 rubyc && chmod +x rubyc
+
+      - run: make build/metanorma
+        env:
+          DEVELOPER_DIR: /Applications/Xcode_11.7.app/Contents/Developer
+
+      - run: build/metanorma help
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: metanorma-macos
+          path: build/metanorma
+
+  macos-test:
+    needs: macos-packed-mn
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.ignore-errors }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ macos-10.15 ]
+        ignore-errors: [ false ]
+        test-flavor:
```

Comment:
I propose to reduce this list just to iso

Reply:
Agree!